pythonforandroid/python.py

```diff
@@ -382,6 +383,22 @@ class HostPythonRecipe(Recipe):
     '''The default url to download our host python recipe. This url will
     change depending on the python version set in attribute :attr:`version`.'''
 
+    @property
+    def python_bin(self):
```

Comment:
I have a vague feeling that it isn't necessarily obvious `python_bin` will be a name only, not a full path. I'd consider making it an internal property (i.e. just call it `_python_bin`). It's a minor thing though.

Reply:
The circumstances it may be different now since the last force push, now we use this property in `PythonRecipe` to determine the right hostpython location (https://github.com/kivy/python-for-android/blob/84adc33df4d32326744ca06357c8aed5d487378d/pythonforandroid/recipe.py#L781). If you still thing that we must set this property private let me know and I will change it, no problem :wink: